lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -191,6 +191,53 @@ struct MemberVisitor {
     return context.convertStatement(procNode.getBody());
   }
 
+  // Handle parameters
+  LogicalResult visit(const slang::ast::ParameterSymbol &paramNode) {
+    auto type = context.convertType(*paramNode.getDeclaredType());
+    if (!type)
+      return failure();
+
+    Value initial;
+    if (const auto *init = paramNode.getInitializer()) {
+      initial = context.convertExpression(*init);
+      if (!initial)
+        return failure();
+    }
+
+    auto namedConstantOp = builder.create<moore::NamedConstantOp>(
+        loc, type, builder.getStringAttr(paramNode.name),
+        paramNode.isLocalParam()
+            ? moore::NamedConstAttr::get(context.getContext(),
+                                         moore::NamedConst::LocalParameter)
+            : moore::NamedConstAttr::get(context.getContext(),
+                                         moore::NamedConst::Parameter),
+        initial);
+    context.valueSymbols.insert(&paramNode, namedConstantOp);
+    return success();
+  }
+
+  // Handle specparam
+  LogicalResult visit(const slang::ast::SpecparamSymbol &spNode) {
+    auto type = context.convertType(*spNode.getDeclaredType());
+    if (!type)
+      return failure();
+
+    Value initial;
+    if (const auto *init = spNode.getInitializer()) {
+      initial = context.convertExpression(*init);
+      if (!initial)
+        return failure();
+    }
```

Comment:
Same as above: you probably want to skip specparams without any value.
```suggestion
    const auto *init = spNode.getInitializer();
    if (!init)
      return success();  // skip parameters without value
    Value initial = context.convertExpression(*init);
    if (!initial)
      return failure();
```